ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/GroupParametersExtension.kt

```diff
@@ -0,0 +1,17 @@
+@file:JvmName("GroupParametersExtension")
+
+package net.corda.v5.ledger.common
+
+import net.corda.v5.membership.GroupParameters
+
+private const val NOTARIES_KEY = "corda.notaries"
+
+/**
+ * A list of all available notary services in the group.
+ * This will parse the available notary services information from the group parameters internal property map.
+ */
+val GroupParameters.notaries: List<NotaryInfo>
```

Comment:
Thinking about this again. I'm not sure we can do this. The coding standards advise against it:
https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4082761773/Public+API+coding+standards#Extension-functions-should-add-to-existing-APIs

We may need to rethink this. Either, we move `NotaryInfo` back to membership, which will mix ledger information into the membership module, or we split the group parameters interface so we have:

```
package net.corda.v5.membership

interface MembershipGroupParameters : LayeredPropertyMap {
    val minimumPlatformVersion: Int
    val modifiedTime: Instant
    val epoch: Int
}
```

```
package net.corda.v5.ledger.common

interface GroupParameters : MembershipGroupParameters {
    val notaries: List<NotaryInfo>
}
```

Open to other suggestions also.

Reply:
I I think that having 2 interfaces can be also confusing to people.

Reply:
Moved notaries under membership module. Having 2 interfaces caused some issues around the `GroupParametersReader`, because that is expected to provide the notary information too, while it must be under membership module in runtime-os.